src/vs/vscode.d.ts

```diff
@@ -6221,11 +6221,16 @@ declare module 'vscode' {
 		export function createInputBox(): InputBox;
 
 		/**
-		 * Create a new [output channel](#OutputChannel) with the given name.
+		 * Creates a new [output channel](#OutputChannel) with the given name.
+		 * By default the UI will not show the appended data immediately.
```

Comment:
Hm, we should sell this better... How about: "Output channels use a separate transport mechanism when appending data. This helps to reduce CPU load on the UI process but also means that the output channel UI updates with a small delay. Use the `force`-flag to enforce immediate updates at the cost of higher CPU usage"

Reply:
I like it. You have better marketing skills 👍 😄